docs/snippets/resource-management/vars.mdx

```diff
@@ -0,0 +1,63 @@
+export const home = {
```

Comment:
Here, we define the links in one place and reuse them when we import the snippets. There may be a way to DRY this file up as well, but I haven't found something that works yet with mdx + URLs.